raiden/tests/unit/functional/test_directtransferlog.py

```diff
@@ -149,7 +149,7 @@ def test_target_log_directransfer_successevent(
     app1_state_changes = get_all_state_events(app1.raiden.transaction_log)
     sucessful_received_transfers = [
         event
-        for _, _, event in app1_state_changes
+        for _, _, _, event in app1_state_changes
```

Comment:
Please, change this to use a attribute of a namedtuple instead of deconstructing the tuple with placeholders. e.g. `state_change.event for state_change in app1_state_changes ...`